docs/manual_doc/Language_Structure/Literal_Values/Numeric_Literals.md

```diff
@@ -0,0 +1,18 @@
+Number literals include interger literals and floating-point literals.
+
+Integers are 64 bit digitals, and can be preceded by + or - to indicate a positive or negative value, respectively. They're the same as `int64_t` in the C language.
+
+floating-points are the same as `double` in the C language.
+
+Here are some examples:
+
+```
+1, -5, +10000100000
+-2.3, +1.00000000000
+```
+
+Notice that the maximum value for the positive interges is `9223372036854775807`. Any value larger than the maximum will be bounded (no over-flow). So as the minimum value `-9223372036854775808` for the negative interges.
```

Comment:
input int value larger than the maximum or less than minimum will be returned syntax error